Update sql_deadlock.md (#26618)
DataDog/documentation: content/en/database_monitoring/guide/sql_deadlock.md

The Deadlock view enables you to explore deadlock events in your SQL Server database.
A deadlock occurs when two or more processes are unable to proceed because each is waiting for the other to release resources.

> 
> This guide assumes that Database Monitoring has been configured for your [SQL Server][1].
> 
> ## Before you begin

Reviewer: ```suggestion

## Before you begin

You must configure Database Monitoring for your [SQL Server][1] before following the steps in this guide.

```

  GO
```

> 2. In addition to the [DBM for SQL Server setup][1]. In the Datadog Agent, enable deadlocks in `sqlserver.d/conf.yaml`.

Reviewer: I would make this clarification in a different place. It belongs in a prerequisites section, not inline in the procedure.

I suggest:

* Get rid of this line change
* Get rid of line 18
* Under line 20 (`## Before you begin`), add a line that says something like `You must configure Database Monitoring for your [SQL Server][1] before following the steps in this guide.`

I've added these as suggested edits.

> 2. In addition to the [DBM for SQL Server setup][1]. In the Datadog Agent, enable deadlocks in `sqlserver.d/conf.yaml`.

Reviewer: ```suggestion
2. In the Datadog Agent, enable deadlocks in `sqlserver.d/conf.yaml`.
```
